Add missing <stdarg.h> include

This is needed to fix the build on uClibc, due to the usage of
va_list.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Index: b/cil/src/cil_log.h
===================================================================
--- a/cil/src/cil_log.h
+++ b/cil/src/cil_log.h
@@ -30,6 +30,7 @@
 #define CIL_LOG_H_
 
 #include <stdlib.h>
+#include <stdarg.h>
 #include <cil/cil.h>
 
 #define MAX_LOG_SIZE 512
